.github/workflows/ci_cd.yml

```
@@ -74,6 +74,56 @@ jobs:
           name: ${{ matrix.image }}
           path: /tmp/${{ matrix.image }}.tar
 
+  build-nginx:
+    # This requires a separate job due to the dependency on the other image builds
+    name: Build `nginx` Dockerfile target
+    runs-on: ubuntu-latest
+    needs:
+      - build-images
+
+    steps:
+      - uses: actions/checkout@v3
+      - uses: extractions/setup-just@v1
+
+      - name: Download all images
+        uses: actions/download-artifact@v2
+        with:
+          path: /tmp
+
+      - name: Load API and ingestion server images
+        run: |
+          docker load --input /tmp/api/api.tar
+          docker load --input /tmp/ingestion_server/ingestion_server.tar
+
+      - name: Start API, ingest and index test data
+        # Is this necessary for collectstatic?
+        run: just up
+
+      - name: collectstatic
+        run: just collectstatic
+
+      - name: Setup Docker Buildx
+        uses: docker/setup-buildx-action@v1
```

Can we include updating this action on this file?
```suggestion
        uses: docker/setup-buildx-action@v2
```